hddm/cnn_models/ornstein_training_data_binned_1_nbins_512_n_100000/checkpoint

```diff
@@ -0,0 +1,2 @@
+model_checkpoint_path: "/media/data_cifs/projects/prj_approx-bayes/projectABC/models/ornstein_training_data_binned_1_nbins_512_n_100000/ornstein_351000.ckpt-351000"
```

Comment:
all these paths are hardcoded, should checkpoints even be here?

Reply:
when the class is initialized its overwritten to the correct checkpoint path for the user. 
But I guess I can find a better solution for this one.

Reply:
fixed this

Reply:
Why not just remove them?

Reply:
done